### ISSUE TITLE
Config does not working

### ISSUE THREAD

**Issue by shersh** [tallakt/codepagex#5]
I'm trying to config using encodings. 
In config.exs:
```
use Mix.Config
  config :codepagex, :encodings, [
  :ascii,           # by alias name
  "VENDORS/MICSFT/WINDOWS/CP1251"
]
```
but when calling 
`iex(1)> Codepagex.encoding_list`
I see
```
["ISO8859/8859-1", "ISO8859/8859-10", "ISO8859/8859-11", "ISO8859/8859-13",
 "ISO8859/8859-14", "ISO8859/8859-15", "ISO8859/8859-16", "ISO8859/8859-2",
 "ISO8859/8859-3", "ISO8859/8859-4", "ISO8859/8859-5", "ISO8859/8859-6",
 "ISO8859/8859-7", "ISO8859/8859-8", "ISO8859/8859-9",
 "VENDORS/MISC/US-ASCII-QUOTES"]

```

So, maybe I have mistake or config is not working?

**Comment by tallakt** [comment]
The config is not being activated. I had a recent issue where code was not recpomiled due to config change. Perhaps you could delete the _build folder and try to recompile? If this works for you I will look into documenting this and if possible find a permanent solution...

**Comment by shersh** [comment]
Yeah, thanks. Now it's working

**Comment by tallakt** [comment]
Ok... Leave the issue open for now.... I will close it myself after fixing it. Thanks!!

**Comment by shersh** [comment]
It's new problem here =(
After restarting Codepage can't find this encoding again

** (MatchError) no match of right hand side value: {:error, "Unknown encoding \"VENDORS/MICSFT/WINDOWS/CP1251\""}

**Comment by tallakt** [comment]
Seems the apostrophes are part of the encoding string... Could you perhaps pm me your code?

**Comment by shersh** [comment]
@tallakt  https://gist.github.com/shersh/d8681ec731ae5243d3d7d86a71d4d215

first time it works nice, but if I Ctrl+C and run this again, error will raise

**Comment by tallakt** [comment]
I's behaving really strange on my computer as well. Sometimes ok, sometimes resulting in error... looking into it...

**Comment by shersh** [comment]
No news?

**Comment by tallakt** [comment]
Im working on it. But it's very tricky. Also busy lately. It _could_ be an elixir bug, so i am determined to figure this one out

Sendt fra min telefon

**Comment by tallakt** [comment]
Fixed in v0.1.3
